shell/platform/embedder/fixtures/main.dart

```diff
@@ -453,3 +453,37 @@ void can_receive_locale_updates() {
   };
   signalNativeTest();
 }
+
+@pragma('vm:entry-point')
+void verify_b143464703() {
```

Comment:
Maybe include a link to a GitHub issue if you're referring a bug, this number is pretty mysterious for non googlers

Reply:
Done. There was no GitHub issue when I began work on this.